cnf-certification-test/accesscontrol/suite.go

```diff
@@ -449,6 +452,12 @@ func TestAutomountServiceToken(env *provider.TestEnvironment) {
 	if n := len(failedPods); n > 0 {
 		logrus.Debugf("Pods that failed automount test: %+v", failedPods)
 		tnf.ClaimFilePrintf("Pods that failed automount test: %+v", failedPods)
-		ginkgo.Fail(fmt.Sprintf("% d pods that failed automount test", n))
+		if !tnf.IsUnitTest() {
```

Comment:
Why not create a ginkgoFail() function that would make the ginkgo call and the if

Reply:
Done.